kinode/src/kernel/standard_host.rs

```diff
@@ -402,6 +402,12 @@ async fn send_and_await_response(
             process.process.metadata.our.process
         ));
     }
+    if t::Address::de_wit(target.clone()) == process.process.metadata.our {
+        return Err(anyhow::anyhow!(
+            "kernel: got invalid send_and_await_response() Request from and to {:?}: cannot await a Request to `our`: will deadlock",
```

Comment:
Personally I would prefer to get the pretty Display print of the `Address` rather than the Debug print

Reply:
Good call; changed in 06c0bd4